examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -9366,7 +9366,7 @@
           "mfgCode": null,
           "define": "ON_OFF_CLUSTER",
           "side": "client",
-          "enabled": 0,
+          "enabled": 1,
```

Comment:
Great question. The reason is that the binding YAML tests tries to write a binding to the on-off cluster and in this case it is mandated by the spec that the client SHALL be available on that endpoint. Previously this was not validated by the SDK, so the test passed because the all-clusters-app would accept the binding entry regardless but now when we actually validate this the test failed, so I had to enable the on-off client on all-clusters-app to make the test pass.